pkgupdate

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+readarray -t lists < <(ls /etc/apt/sources.list.d/*)
+pkgs=""
+for list in "${lists[@]}"
+do
+	list=$(basename "${list}")
+	if [[ "${list}" != "ubports.list" && "${list}" != "ubports-android9.list" ]]; then
+		list="${list/ubports-/}"
+		pkgs=$(pkgcheck "${list/.list/}")" "${pkgs}
+	 fi
+done
```

Comment:
same as before, you don't need to use an array variable here if you just read the packages line by line.

Reply:
is that the correct line?

Reply:
oh now i get it